doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.loadReferenceCatalog.LoadReferenceCatalogTask.rst

```diff
@@ -0,0 +1,42 @@
+.. lsst-task-topic:: lsst.pipe.tasks.loadReferenceCatalog.LoadReferenceCatalogTask
+
+########################
+LoadReferenceCatalogTask
+########################
+
+``LoadReferenceCatalogTask`` is a convenience task that combines the reference object loader adapted for multiple filters; color term application; and reference selection in one task.
+This task returns a numpy record array with the magnitude information for each of the physical filters requested by the caller.
+
+If appropriate, all proper motion corrections are handled by the reference object loader as configured.
+Mapping from physical filter to reference filter is handled by the ``filterMap`` configured with the reference object loader, as well as the color term library.
+
+The format of the reference catalogs returned by this task will be a numpy record array with the following datatype:
+
+.. code-block:: python
+
+    import numpy as np
+    dtype = [('ra', 'np.float64'),
+             ('dec', 'np.float64'),
+             ('refMag', 'np.float32', (len(filterList), )),
+             ('refMagErr', 'np.float32', (len(filterList), ))]
+
+Reference magnitudes (AB) will be 99 for non-detections for a given reference filter.
```

Comment:
Reference magnitude errors will also be set to this sentinel value too, right? Would it be helpful to mention that explicitly here?